cuenca/resources/card_transactions.py

```diff
@@ -0,0 +1,35 @@
+import datetime as dt
+from dataclasses import dataclass
+from typing import ClassVar, List, cast
+
+from cuenca_validations.types import (
+    CardErrorType,
+    CardNetwork,
+    CardTransactionType,
+    CardType,
+)
+
+from .base import Transaction
+from .resources import retrieve_uri
+
+
+@dataclass
+class CardTransaction(Transaction):
+    _resource: ClassVar = 'card_transactions'
+
+    type: CardTransactionType
+    network: CardNetwork
+    related_card_transaction_uris: List[str]
+    card_last4: str
+    card_type: CardType
+    merchant: str
+    metadata: dict
+    error_type: CardErrorType
+    expired_at: dt.datetime
```

Comment:
The better way to handle expirations is to make `expiration` a card transaction type and create a new transaction

Reply:
Having `expired_at` would be like adding `voided_at` for voids. It does make sense since it only applies to auths